frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1402,6 +1407,86 @@ public void update(ActionRequest request, ActionResponse response) throws Portle
         }
     }
 
+    private RequestStatus updateLinkedObligations(ActionRequest request, Project project, User user, ProjectService.Iface client) {
+        try {
+            final JsonNode rootNode = OBJECT_MAPPER.readTree(request.getParameter(OBLIGATION_DATA));
+            final boolean isDeleteAllOrphanObligations = Boolean.valueOf(request.getParameter(DELETE_ALL_ORPHAN_OBLIGATIONS));
+            final boolean isObligationPresent = CommonUtils.isNotNullEmptyOrWhitespace(project.getLinkedObligationId());
+            final String email = user.getEmail();
+            final String createdOn = SW360Utils.getCreatedOn();
+            ProjectObligation obligation;
+
+            if (isObligationPresent) {
```

Comment:
May be we can use ternary operator here.

Reply:
Used ternary operator.